roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -78,39 +91,11 @@ subjects:
   name: kubernetes-dashboard
   namespace: {{ system_namespace }}
 
----
-# ------------------- Gross Hack For anonymous auth through api proxy ------------------- #
```

Comment:
Unfortunately we still need this block to allow UI access through the proxy without having to use kubectl proxy.

Reply:
ok